meta/meta.go

```diff
@@ -196,52 +175,34 @@ func (m *Meta) GenSchemaVersion() (int64, error) {
 
 func (m *Meta) checkDBExists(dbKey []byte) error {
 	v, err := m.txn.HGet(mDBs, dbKey)
-	if err != nil {
-		return errors.Trace(err)
-	} else if v == nil {
-		return ErrDBNotExists
+	if err == nil && v == nil {
+		err = ErrDBNotExists
 	}
-
-	return nil
+	return errors.Trace(err)
```

Comment:
no need to trace

Reply:
It's original code,  and other cases in this file all use the trace. I think removing trace can be in other PR.